plugin-server/src/config/config.ts

```diff
@@ -74,6 +74,7 @@ export function getDefaultConfig(): PluginsServerConfig {
         JOB_QUEUE_GRAPHILE_URL: '',
         JOB_QUEUE_GRAPHILE_SCHEMA: 'graphile_worker',
         JOB_QUEUE_GRAPHILE_PREPARED_STATEMENTS: false,
+        JOB_QUEUE_GRAPHILE_CONCURRENCY: 1,
```

Comment:
May as well increase the default, 1 seems _very_ low.

Reply:
I prefer to decouple code and config changes, let's go back to that default once it's proven fine on prod